deploy/tasks/main.yaml

```diff
@@ -20,11 +20,13 @@
   file:
     path: '{{ deploy_helper.new_release_path }}'
     state: directory
+    mode: '0755'
```

Comment:
Si je comprend bien on peut aussi écrire `mode: 0755` ou `mode: '755'`.
Mais écrit comme tu l'as fait au moins on ne se pose pas la question :smile:

Reply:
Ho, you are right, I've missread the documentation.

Reply:
actually all three forms should work well, as long as it's read as a string and not an int. Or you could also use the decimal representation... :trollface: